src/v/kafka/server/handlers/topics/types.h

```diff
@@ -81,7 +81,6 @@ static constexpr std::array<std::string_view, 21> allowlist_topic_noop_confs = {
   "min.cleanable.dirty.ratio",
   "message.timestamp.difference.max.ms",
   "message.format.version",
-  "max.message.bytes",
```

Comment:
This comment is unrelated to this exact line, but should we also be modifying the topic properties in the `supported_configs` set in `create_topics.cc`?

I remember when implementing KIP-525 a while back that there were topic properties returned from create topic requests at version >= 5.

Reply:
good point, let me verify this